tx/tx_dynamic_fee.go

```diff
@@ -86,14 +86,14 @@ func (t *dynamicFeeTransaction) gasPriceCoef() uint8 {
 
 func (t *dynamicFeeTransaction) maxFeePerGas() *big.Int {
 	if t.MaxFeePerGas == nil {
-		return nil
+		return common.Big0
```

Comment:
I would check places where we are checking against `nil` (I have seen some for `maxPriorityFeePerGas()`.

For example:

https://github.com/vechain/thor/blob/release/galactica/runtime/resolved_tx.go#L51
https://github.com/vechain/thor/blob/release/galactica/txpool/validation.go#L37

Reply:
Yes, they're probably useless now. I'll remove them. Thanks